internal/output/ghworkflow/types.go

```diff
@@ -23,10 +23,11 @@ type Concurrency struct {
 
 // On represents GitHub Actions event triggers.
 type On struct {
-	Push        `yaml:"push,omitempty"`
-	PullRequest `yaml:"pull_request,omitempty"`
-	Schedule    []Schedule `yaml:"schedule,omitempty"`
-	WorkFlowRun `yaml:"workflow_run,omitempty"`
+	Push             `yaml:"push,omitempty"`
+	PullRequest      `yaml:"pull_request,omitempty"`
+	Schedule         []Schedule `yaml:"schedule,omitempty"`
+	WorkFlowRun      `yaml:"workflow_run,omitempty"`
+	WorkFlowDispatch `yaml:"workflow_dispatch"` // no omitempty, apply to all
```

Comment:
probably make it a bit more explicit, and `omitempty` actually and enable it e.g. for the default workflow for fully Kres-managed projects, and for custom projects make it configurable?

Reply:
perhaps this would need an empty struct containing only a dummy ignored field

Reply:
https://github.com/siderolabs/kres/blob/main/internal/project/auto/config.go#L37 i think we can enable based on a field here